R/runfiles_commands.bzl

```diff
@@ -0,0 +1,58 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
You can have this copyright to be for "Bazel R Rules Authors", and change the year to 2018.